### PR TITLE
Integration of functions for plots

### LINKED ISSUE #4: Functions for plots
TODO: create functions to provide the same plots of the original [SpatialDE python package](https://github.com/Teichlab/SpatialDE).

Example plots from:
- main page of [SpatialDE python package](https://github.com/Teichlab/SpatialDE) 
- workflow analysis of [MouseOB dataset](https://github.com/Teichlab/SpatialDE/blob/master/Analysis/MouseOB/Mouse%20Olfactory%20Bulb%20Analysis.ipynb)

### PR DESCRIPTION
**Pull request by davidecrs**
Added functions for plots. 
Thanks to  @lambdamoses

- `FSV_sig`: Plot Fraction Spatial Variance vs Q-value
- `multiGenePlots`: Plot Spatial Patterns of Multiple Genes. Integrated with S4 method for `matrix` and `SpatialExperiment` object

Close #4

### CODE REVIEW

File: R/plots.R [hunk 1]
```diff
@@ -0,0 +1,228 @@
+
+#' Plot Fraction Spatial Variance vs Q-value
```
Comment (milanmlft) [inline R/plots.R:1]: We should provide a small example in the documentation of this function

**Review by milanmlft — CHANGES_REQUESTED** [review]
This looks good to me! I will just push some small updates to format the code in "BioConductor" style and some small improvements to the vignette.

The only thing that's missing is an example in the documentation of `FSV_sig` I think.

**Comment by lambdamoses** [comment]
I wonder if the data wrangling should be rewritten with base R to get rid of the dplyr and magrittr dependencies, though I trust that dplyr and magrittr are very unlikely to introduce breaking changes.

**Review by lambdamoses — APPROVED** [review]

**Comment by milanmlft** [comment]
> I wonder if the data wrangling should be rewritten with base R to get rid of the dplyr and magrittr dependencies, though I trust that dplyr and magrittr are very unlikely to introduce breaking changes.

I agree, they're only used in a few places, so removing them as dependency would make sense.

**Review by milanmlft — APPROVED** [review]

**Review by lambdamoses — COMMENTED** [review]
I'll rewrite the Tidyverse code with base R.

**Pull request merged**